builtin/providers/aws/import_aws_iam_policy_test.go

```diff
@@ -0,0 +1,85 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/iam"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+var testAccAwsIamPolicyConfig = `
+resource "aws_iam_policy" "test" {
+    name = "test_policy"
```

Comment:
We need to randomize this - if this fails then the test will fail the following night due to orphaned resources. This should be a func that accepts a random string as a name